### PR TITLE
Integration test search term

### PR DESCRIPTION
**Pull request by Pawlette**
Create an integration test for searching a term

https://trello.com/c/acCDsCMR/18-integration-test-for-searching-a-term

### CODE REVIEW

File: client/src/__mocks__/Auth.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default class Auth {
+    isAuthenticated() { } 
+  }
```
Comment (jcholder-thoughtworks) [inline client/src/__mocks__/Auth.js]: Minor whitespace issue here and above.

File: client/tests/SearchTerm.integ.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const resultCardDescriptionContainer = await waitForElement(() => getByText(/I define it as a sort of layer gender/))
+
+    // Arrange
+    const resultCard = container.querySelector('.result-card')
+    const termHeaderInResultCard = resultCard.querySelector('h5')
+
+    // Assert
+    expect(termHeaderInResultCard.innerHTML).toBe('abrogender')
+    expect(resultCard).toContainElement(resultCardDescriptionContainer)
+  });
```
Comment (jcholder-thoughtworks) [inline client/tests/SearchTerm.integ.test.js]: Indentation issue inside the `it` block.

File: client/tests/SearchTerm.integ.test.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      history,
+    }
+  }
+
+it('Renders main page and searches for a term', async () => {
+    // Arrange
+    const defaultQueries = customQueries.default
+    const { getByTestId, getByText, history, container } = renderWithRouter(<App auth={auth} term="" />,
+    { queries: { ...queries, ...defaultQueries }, })
+    const input_for_SearchTerm = getByTestId('mainInputSearchTerm')
```
Comment (jcholder-thoughtworks) [inline client/tests/SearchTerm.integ.test.js]: Should this be only camelCase instead of mixed case?

**Comment by jcholder-thoughtworks** [comment]
Aside from the minor formatting comments, looks good to me!

File: client/tests/SearchTerm.integ.test.js (hunk 3 above)
Comment (Pawlette) [reply client/tests/SearchTerm.integ.test.js]: yes, camelCase definitely, I don't know why I used snake XD. I'll change it to coordinate with the other variables :)

File: client/tests/SearchTerm.integ.test.js (hunk 2 above)
Comment (Pawlette) [reply client/tests/SearchTerm.integ.test.js]: hehehe.. i'll adjust it 👍

File: client/src/__mocks__/Auth.js (hunk 1 above)
Comment (Pawlette) [reply client/src/__mocks__/Auth.js]: Thank you! I'll fix it :)

**Pull request merged**